lib/components/atoms/Video/tests/Video.test.js

```diff
@@ -0,0 +1,34 @@
+import { shallow, render } from 'enzyme';
+import { VideoVanilla } from '../index';
+import defaultConfig from '../Video.mock';
+
+describe('<VideoVanilla />', () => {
+  let VideoComponent = '';
+  beforeEach(() => {
+    VideoComponent = shallow(<VideoVanilla {...defaultConfig} />);
+  });
+
+  it('should have video tag', () => {
+    expect(VideoComponent.find('video')).toBeTruthy();
+  });
+
+  it('should have source tag', () => {
+    expect(VideoComponent.find('video').find('source')).toBeTruthy();
+  });
+
+  it('should have src attribute in source tag', () => {
+    const VideoComponentRender = render(<VideoVanilla {...defaultConfig} />);
+    expect(VideoComponentRender.find('source').attr('src')).toEqual(defaultConfig.src);
+  });
+
+  it('should be autoplay but need to be mute', () => {
```

Comment:
There could be a situation where the config passed had { autoplay: false }, do test that behaviour too